test/ILLink.Tasks.Tests/ILLink.Tasks.Tests.cs

```diff
@@ -306,6 +302,40 @@ public void TestWarn (string warnArg, WarnVersion expectedVersion)
 			}
 		}
 
+		[Theory]
+		[InlineData ("", "blah", true, new uint[] { }, new uint[] { })]
```

Comment:
Just FYI, when invoked from MSBuild targets, I think that the task will get a `null` (rather than an empty string) if the task parameter is not set or if it is set to an empty MSBuild property.